bundles/target/src/main/java/org/jscsi/target/TargetServer.java

```diff
@@ -244,7 +274,7 @@ public Void call () throws Exception {
 
                     sessions.add(session);
                     // threadPool.submit(connection);// ignore returned Future
-                    connection.call();
+                    new Thread(new ConnectionHandler((newConnection))).start();

```

Comment:
Heavyweight to create a new `Thread` per request.  Use an `ExecutorService` instead?